src/components/diagrams/diagram-pane.js

```diff
@@ -1098,7 +1097,6 @@ export function DiagramPane({
                                     diagramView.svgType ===
                                         DiagramType.SUBSTATION) && (
                                     <SingleLineDiagramContent
```

Comment:
Here you need the loadflow status to set the flows in disable or not

Reply:
Done